src/components/tooltip/tooltip.js

```diff
@@ -51,6 +59,14 @@ class Tooltip extends React.Component {
         }
         return null;
       },
+      PopperWrapperComponent: (props, propName) => {
```

Comment:
🤔 aren't we leaking implementation details with this name? The fact that we use popper?

Reply:
🤔 yeah makes sense.

How about `TooltipWrapperComponent`?